pkg/utils/encrypt.go

```diff
@@ -43,3 +43,12 @@ func Decrypt(ciphertextB64 string) (string, error) {
 	}
 	return string(plaintext), nil
 }
+
+// DecryptOrPlaintext tries to decrypt base64 encoded ciphertext to plaintext or return plaintext
+func DecryptOrPlaintext(ciphertextB64 string) (string, error) {
```

Comment:
seems that dosen't need to return error

Reply:
ok, i have cleaned the extra `err` variables